app/models/user.rb

```diff
@@ -57,6 +60,10 @@ def new_account(request, email)
       @user = User.new(email: email, name: full_name, activated: true, password: password)
       @user.save
     end
+
+    def already_liked?(user, micropost_id)
```

Comment:
hàm này sao lai cho vào self nhỉ?
hàm này phải la hàm củ instance chứ ko phải hàm của class
chưa kể hàm này nên viết ở bên model react hơn, tránh làm phingf to model User